### DISCUSSION TITLE
ImportError: cannot import name 'Expression_T' from 'nonebot.typing' (D:\Python\IDLE\lib\site-packages\nonebot\typing.py)

### DISCUSSION THREAD

**Discussion by Phantom-X** [nonebot/discussions#73] [问答]
from .typing import Expression_T, PermissionPolicy_T
ImportError: cannot import name 'Expression_T' from 'nonebot.typing' (D:\Python\IDLE\lib\site-packages\nonebot\typing.py)

**Comment by yanyongyu** [comment]
请勿同时安装nonebot1和nonebot2哦
